CONTRIBUTING.md

```diff
@@ -174,6 +162,23 @@ We use [asciidoc](https://asciidoctor.org/docs/what-is-asciidoc/) to write the d
 
 An [AsciiDoc IntelliJ Plugin](https://plugins.jetbrains.com/plugin/7391-asciidoc) is a helpful plugin that permits visualizing .adoc files directly in IntelliJ
 
+##### VS Code
```

Comment:
:question: could we move all IDE tips in a dedicated page in the new docs/dev folder?
The contributing guide is very large and it's currently hard to follow how to contribute vs how to develop (this is also related to contribution but this this more details/setup)

Reply:
thx